pkg/network/ebpf/c/protocols/protocol-classification-defs.h

```diff
@@ -15,6 +15,32 @@
 // The minimal HTTP request has 16 characters: GET x HTTP/1.1\r\n
 #define HTTP_MIN_SIZE 16
 
+// Postgres
+
+#define POSTGRES_MIN_MSG_SIZE 5
+// From https://www.postgresql.org/docs/current/protocol-overview.html:
+// The first byte of a message identifies the message type, and the next four bytes specify the length of the rest
+// of the message (this length count includes itself, but not the message-type byte). The remaining contents of the
+// message are determined by the message type.
+// The minimum payloads are for the MOVE/COPY messages which are 4 bytes, so the minimum size is 8 bytes (4 bytes for
+// message len, and 4 bytes for the smallest payload).
+#define POSTGRES_MIN_PAYLOAD_LEN 8
```

Comment:
Wondering if we should just define 

```c
#define POSTGRES_MIN_MSG_SIZE 13
```

Reply:
That would be the minimum size of a message with a payload, but not all messages have a payload. You can find them described here: https://www.postgresql.org/docs/current/protocol-message-formats.html

For example, the `CloseComplete` message:
```
CloseComplete (B)

    Byte1('3')

        Identifies the message as a Close-complete indicator.
    Int32(4)

        Length of message contents in bytes, including self.
```